### PR TITLE
[zh] updated two files in content/zh/docs/contribute/

### PR DESCRIPTION
**Pull request by windsonsea**
Added translations and correct minor errors. This submission is batch 3 of #33474 from @tengqm.

### CODE REVIEW

File: content/zh/docs/contribute/review/reviewing-prs.md [hunk 1]
```diff
@@ -55,19 +55,59 @@ Before you start a review:
...
+## 评阅过程  {#review-process}
+
+一般而言，应该使用英语来评阅 PR 的内容和样式。Figure 1 outlines the steps for the review process. The details for each step follow.
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
+    subgraph fourth[开始评阅]
```
[the remark this answers is not in the document]
Comment (windsonsea) [reply content/zh/docs/contribute/review/reviewing-prs.md:81]: 这的确是个问题，每个 topic 翻的不一致，只能保证一个 topic 内的一致性。评阅、评审、审查...

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Comment by windsonsea** [comment]
> Thanks. /lgtm /approve

Thanks, Qiming. Could I ask you give me a favor +1 when I apply for a member title of sig-docs. Thanks for your patience and kindness to help me successfully complete those 5 PRs.

**Comment by tengqm** [comment]
@windsonsea I'd be more than happy to sponsor anyone who are passionate in community contributions and who are devoted to long term sustainability of the docs website.

**Pull request merged**